go.mod

```diff
@@ -88,7 +89,6 @@ require (
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
 	github.com/armon/go-metrics v0.4.0 // indirect
 	github.com/asaskevich/govalidator v0.0.0-20210307081110-f21760c49a8d // indirect
-	github.com/aws/aws-sdk-go v1.44.187 // indirect
```

Comment:
Is the version bump needed for this change? It seems unrelated (but I might be missing something). If the version bump is not required, could you please change this PR to use the previous version?

Reply:
the thing is when I ran the _go mod vendor_ this was the output to the file

Reply:
I ran it because of the new import

Reply:
> Is the version bump needed for this change? It seems unrelated (but I might be missing something). If the version bump is not required, could you please change this PR to use the previous version?

The thing is when I ran the _go mod vendor_ because of the new import, this was the output to the file

Reply:
Have you tried editing `go.mod` to use the same version as before?